[ADD] sign: add aadhar esign documentation (#14769)
odoo/documentation: content/applications/productivity/sign.rst

clicking on :guilabel:`Validate & Send Completed Document`, triggering a
:guilabel:`Final verification` page where authentication via itsme® is required.

.. _sign/Aadhaar:

Aadhaar eSign
~~~~~~~~~~~~~

Aadhaar eSign allows signers in India to digitally sign documents using their Aadhaar number and
:abbr:`OTP (One Time Password)` verification. This provides a secure and legally valid way to complete signatures directly
within Odoo Sign.

To enable Aadhar eSign go to :menuselection:`Sign -> Configuration -> Settings` and toggle
on :icon:`fa-toggle-on` :guilabel:`Sign with Aadhar eSign`.

Request signer authentication through Aadhaar eSign

Once you’ve uploaded your document, click the :icon:`fa-ellipsis-vvertical` ellipsis on the signer’s box you want to
authenticate. Click :icon:`fa-pencil-square-o` :guilabel:`edit`, then, in the Signer Edition pop-up, on the

> :guilabel:`Authentication` field, select :guilabel:`Aadhaar eSign`. Click :guilabel:`Save`.

Reviewer: ```suggestion
To request signer authentication through Aadhaar eSign:

- Upload or edit an existing PDF.
- On the left panel, click the :icon:`fa-ellipsis-vertical` :guilabel:`(vertical ellipsis)` icon next to the signer that should authenticate with Aadhaar eSign and click :icon:`fa-pencil-square-o` :guilabel:`Edit`.
- In the pop-up, select :guilabel:`Via Aadhaar eSign` under :guilabel:`Authentication` and click :guilabel:`Save`.
```
I tried to clarify the steps using bullet points (my comments don't respect the 100-character limit)

Sign with Aadhaar eSign

Upon signing the document, the signer completes the :guilabel:`Signature` field and proceeds by clicking
:guilabel:`Validate & Send Completed Document`, triggering a :guilabel:`Final verification` page where authentication

> via Aadhaar is required.

Reviewer: ```suggestion
Upon signing the document, an extra :guilabel:`Final verification` page will be displayed where authentication via Aadhaar is required.
```
This isn't really necessary, but it should at least be shortened.

clicking on :guilabel:`Validate & Send Completed Document`, triggering a
:guilabel:`Final verification` page where authentication via itsme® is required.

.. _sign/Aadhaar:

Aadhaar eSign
~~~~~~~~~~~~~

> Aadhaar eSign allows signers in India to digitally sign documents using their Aadhaar number and OTP

Reviewer 2: Hello @nikj-odoo, thank you for your work!

I adapted the text according to the guidelines provided in the documentation. Unless necessary, we tend to avoid adding screenshots due to maintenance updates between versions.

verification. This provides a **Secure and legally valid** way to complete signatures directly within Odoo
Sign.

Aadhar eSign can be enabled from the Configuration screen of the Sign App.

.. image:: sign/aadhaar-configration.png
   :alt: install Aadhaar eSign

> :guilabel:`Request Signer authentication through Aadhaar eSign`

Reviewer 2: Request signer authentication through Aadhaar eSign
*****************************************************

Once you’ve uploaded your document, click the :icon:`fa-ellipsis-v`vertical ellipsis on the signer’s box you want to authenticate. Click :icon:`fa-pencil-square-o` :guilabel:`edit`, then, in the Signer Edition pop-up, on the :guilabel: `Authentication`field, select :guilabel:`Aadhaar eSign`. Click :guilabel:`Save`.

Once you’ve uploaded your document, click the **Three dot menu**, then **Signer Edition**, and select
**Aadhaar eSign** in the **Authentication** field.

.. image:: sign/aadhaar-identification.png
   :alt: select customer identified with Aadhaar eSign

:guilabel:`Sign with Aadhaar eSign`

As the signer, complete the signature and click on the Validate button. You will be redirected to the
eMudhra portal (government-certified authority), where Signers must enter their **Aadhaar number** and
authenticate themselves with a **one-time password (OTP)** sent to the phone number associated with their
Aadhaar number.

.. image:: sign/aadhaar-otp-verification.png
   :alt: Aadhaar OTP verification

After entering the OTP, signers can complete the e-signing process and finalize the document.

In the downloaded document, users will see the **Attached certification** from eMudhra.
This certification serves as a **Legally valid proof**, issued by a government-authorized
eSign provider, confirming that the document was signed by the correct Aadhaar holder.

> 

Reviewer 2: Sign with Aadhaar eSign
************************

Upon signing the document, the signer completes the :guilabel:`Signature` field and proceeds by clicking :guilabel:`Validate & Send Completed Document`, triggering a :guilabel:`Final verification` page where authentication via Aadhaar is required.

.. note::
   The digital certification from eMudhra is available in the downloaded document.

clicking on :guilabel:`Validate & Send Completed Document`, triggering a
:guilabel:`Final verification` page where authentication via itsme® is required.

.. _sign/Aadhaar:

Aadhaar eSign
~~~~~~~~~~~~~

Aadhaar eSign allows signers in India to digitally sign documents using their Aadhaar number
and OTP verification. This provides a **secure and legally valid** way to complete signatures
directly within Odoo Sign.

> You can enable feature Aadhar sign from the **sign configuration.**

Reviewer 3: ```suggestion
You can enable feature Aadhaar eSign from the **sign configuration.**
```

.. image:: sign/aadhar-configration.png
   :alt: install Aadhaar eSign

:guilabel:`Configure Aadhaar eSign for a Signer`

Upload the document that needs to be signed. Add a signer by clicking **Add** in the

> **Signers** section.In the **Signer Edition** dialog (see image below), select

Reviewer 3: ```suggestion
**Signers** section. In the **Signer Edition** dialog (see image below), select
```
